.circleci/config.yml

```diff
@@ -23,6 +23,7 @@ version: 2
 # Run static code analyzer for SourceCloud
 # Upload test results
 # Every run command should start with source ${INIT_ENV} to ensure all default dependancies are available
+# TODO Replace with GitHub action to remove branch coupling (develop/release-candidate/master)
```

Comment:
I believe we've started adding JIRA ticket numbers in these `TODO` comments. Can we add this JIRA story here as well?

Reply:
We've actually been adding it directly to the TODO comment recently (at least in CARMA Platform-related repos), like `# TODO (CAR-####) ...` Could we make that change here (and to the other TODO comment as well)?

We won't have this on the sync PRs that have already been merged, but I think we should add this to the other open PRs as well.